docs/installation/local.md

```diff
@@ -1,5 +1,24 @@
 # Installing Loki Locally
 
+# RPM packages for Linux distros (openSUSE Linux)
+
+If you use openSUSE Linux and RPM based distros, you can install packaged version of Loki with:
```

Comment:
@rfratto  yes. For fedora or centos user, they could install the pkgs with RPM ways. We have also some repository for centos7 and fedora but this are more community driven.

I didn't mention other pkgs manager since an user would need to user their own way for doing it.
Perhaps in future we will provide centos7 and other distro pkgs from the same project. It would however require some help from other people. 

The openSUSE pkg will be maintained but for others I can't guarantee

Reply:
For now, I think we should remove the mention of other RPM based distributions and just leave this specific to openSUSE.

Reply:
make sense done